frontend/packages/noobaa-storage-plugin/src/components/capacity-breakdown/capacity-breakdown-card.tsx

```diff
@@ -191,7 +195,14 @@ const BreakdownCard: React.FC = () => {
   return (
     <DashboardCard>
       <DashboardCardHeader>
-        <DashboardCardTitle>{t('noobaa-storage-plugin~Capacity breakdown')}</DashboardCardTitle>
+        <DashboardCardTitle>
+          {t('noobaa-storage-plugin~Capacity breakdown')}
+          <FieldLevelHelp>
+            {t(
+              'noobaa-storage-plugin~This card shows used capacity by different resources. The available capacity is based on cloud services therefore it cannot be shown.',
+            )}
```

Comment:
Lets be consistent, We are not mentioning card anywhere.
```suggestion
              'noobaa-storage-plugin~Shows used capacity by different resources. The available capacity is based on cloud services therefore it cannot be shown.',
            )}
```